src/controls/dynamicForm/DynamicForm.tsx

```diff
@@ -597,8 +597,8 @@ export class DynamicForm extends React.Component<
             hiddenName = response.value;
             termSetId = field.TermSetId;
             anchorId = field.AnchorId;
-            if (item !== null) {
-              item[field.InternalName].forEach((element) => {
+            if (item !== null && item[field.InternalName] !== null && item[field.InternalName].results !== null) {
```

Comment:
This update shouldn't be relevant once your branch wull be updated with latest commits on dev